admin/build/createpackage

```diff
@@ -1,85 +0,0 @@
-#!/bin/bash
```

Comment:
It works. But it's configuration was moved to the more standard `/etc/piezo-admin/conf`.

That will have to be changed in Lucid's Chef configuration. I have not done that.